src/components/forms/deliveryDriver/DeliveryDriverForm.tsx

```diff
@@ -25,17 +25,8 @@ interface PropsDeliveryBox{
   onChanged ?: (value:DeliveryDriver) => void
 }
 
-export default function DeliveryDriverFormBox(props:PropsDeliveryBox) {
-  const fields = [
-    "taxId",
-    "businessName",
-    "denomination",
-    "registeredOffice",
-    "fiscalCode",
-    "pec",
-    "phoneNumber",
-    "uniqueCode"
-  ];
+export function DeliveryDriverForm(props:PropsDeliveryBox) {
+  const fields = ["taxId", "businessName", "denomination", "registeredOffice", "fiscalCode", "pec", "phoneNumber", "uniqueCode"];

```

Comment:
@CriMDev97 this looks like a repetition of keys defined in `fields.ts` for `fieldsDriver`.
You can use const assertions in fields.ts file and declare types accordingly, eg.
```
const fields = ["taxId", "businessName", ... "] as const;
export type FieldTypes = typeof fields[number];
export type FieldsDriver = Record<FieldTypes, FieldsProps>;
export const fieldsDriver: FieldsDriver = ...
```